### PR TITLE
💄widget style update

### PR DESCRIPTION
**Pull request by theixdev**
@tiagov8  @JackDevAU 

Updated Rules Widget Stylings to match video at 
https://www.youtube.com/watch?v=CJPD3eJR0PA

### CODE REVIEW

**Comment by theixdev** [comment]
image

File: src/lib/components/Widget.js [hunk 1]
```diff
@@ -67,11 +67,11 @@ class Widget extends React.Component {
             <img src={Logo} alt="SSW Logo" height="60" width="130"></img>
           </a>
         ) : ''}
-          <h1>
+          <h4>
             <a rel="noreferrer" target="_blank" href={`${this.sswUrl}/rules`}>
```
Comment (JackDevAU) [inline src/lib/components/Widget.js:71]: Might need to add a check to make sure that this doesn't link to the same page. (If this is on /rules it shouldn't link to /rules) as per https://www.ssw.com.au/rules/do-you-avoid-linking-a-page-to-itself

**Review by JackDevAU — CHANGES_REQUESTED** [review]

File: src/lib/components/Widget.js (hunk 1 above)
Comment (theixdev) [reply src/lib/components/Widget.js:71]: That should be ok because the rules widget only shows on pages that it can't link to (i.e People/Category)

**Comment by theixdev** [comment]
image

File: src/lib/components/Widget.js [hunk 2]
```diff
@@ -67,11 +67,11 @@ class Widget extends React.Component {
...
               Latest Rules
```
Comment (JackDevAU) [inline src/lib/components/Widget.js:72]: oops, this was the one that I meant. 
Might need to add a check to make sure that this doesn't link to the same page. (If this is on /rules it shouldn't link to /rules) as per https://www.ssw.com.au/rules/do-you-avoid-linking-a-page-to-itself
image
Figure: Clicking on "latest rules" links to "https://www.ssw.com.au/rules/"

**Review by JackDevAU — CHANGES_REQUESTED** [review]

**Comment by theixdev** [comment]
widget

**Review by JackDevAU — APPROVED** [review]

**Pull request merged**